ccvm_simulators/ccvmplotlib/tests/test_BoxQP_metadata.py

```diff
@@ -35,52 +49,69 @@ def test_BoxQP_metadata_invalid_problem_type(self):
         with self.assertRaises(ValueError):
             BoxQPMetadata(
                 ProblemType(self.invalid_problem_type_str),
-                self.valid_TTS_type,
             )
 
-    def test_BoxQP_metadata_invalid_TTS_type(self):
-        """Test BoxQP Metadata class object when an invalid TTS type is
-        given.
-
-
-        """
-        with self.assertRaises(ValueError):
-            BoxQPMetadata(self.valid_problem_type, TTSType(self.invalid_TTS_type_str))
-
     def test_ingest_result_data_method_valid(self):
         """Test BoxQP Metadata class "ingest_result_data" method when valid
         inputs are given.
 
         # TODO: Implementation
         """
-        pass
+        boxqp_metadata = BoxQPMetadata(self.valid_problem_type)
+        boxqp_metadata.ingest_metadata(self.valid_metadata_filepath)
+
+        self.assertGreater(boxqp_metadata._BoxQPMetadata__batch_size, 0)
+        self.assertGreater(len(boxqp_metadata._BoxQPMetadata__problem_size_list), 0)
```

Comment:
Will these values be the same every time, since we are using some set metadata? If so, let's check the exact value.